TYPO3.Media/Classes/TYPO3/Media/Validator/ImageOrientationValidator.php

```diff
@@ -22,7 +22,7 @@
  *
  * *Example*::
  *
- *   [at]Flow\Validate("$image", type="\TYPO3\Media\Validator\ImageOrientationValidator",
+ *   [at]Flow\Validate("$image", type="\Neos\Media\Validator\ImageOrientationValidator",
```

Comment:
Even though this is just a comment, it will lead people to the use of a not (yet) existing class. So, please only merge with the actual vendor name change.